share/wake/lib/system/plan_scorer.wake

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 SiFive, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You should have received a copy of LICENSE.Apache2 along with
+# this software. If not, you may obtain a copy at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package plan_scorer
```

Comment:
Is there anything left in the main package which uses the `topic runner`?  If not we should move that definition here as well.  Otherwise, calling this `plan_scorer` seems a bit internal-details-y.

Reply:
Ah I forgot about the topic

Reply:
I had just blindly assumed it had moved over! Thanks Sam!